### ISSUE TITLE
[TASK] Implement JSON Viewer to show and copy values 

### ISSUE THREAD

**Issue by johbaxter** [SEMOSS/semoss-ui#263]
### Description 

We need a better UI to view properties on the JSON tied to our notebook queries and variables as well as our blocks

### Existing

https://viewer.textea.io/

### Reference

This issue references. . .

### Tasks

- [ ] Add package from above, for our JSON variables on our VariablePreview component
- [ ] Add package from above, for Notebook cells if the Output is JSON

**Comment by JoshRome26** [comment]
Looking to have this done by September 25th. Will message if any blockers / progress comes up.

**Comment by JoshRome26** [comment]
Running code cell that console prints json:
![image](https://github.com/user-attachments/assets/ca9fdc94-a92e-4f6b-a711-27657f2d539e)

Running data import / import; console prints json: 
![image](https://github.com/user-attachments/assets/303bedeb-a72f-45a3-a083-f83e61f3f49b)

Create Variable preview:
![image](https://github.com/user-attachments/assets/fa979d3f-b540-4f65-9846-204b6c1190bd)

**Comment by JoshRome26** [comment]
Do we want the variable preview when hovering over a json / array variable to also have this viewer ?
![image](https://github.com/user-attachments/assets/1d160fa2-8cf9-40ae-b083-243f0587abe7)

**Comment by johbaxter** [comment]
Yup exactly like that screenshot.  For JSON and Arrays you can likely hide that value at the bottom

**Comment by johbaxter** [comment]
Completed
